cmd/image-builder/main.go

```diff
@@ -520,12 +520,19 @@ operating systems like Fedora, CentOS and RHEL with easy customizations support.
 
 	rootCmd.AddCommand(describeImgCmd)
 
+	verbose, err := rootCmd.Flags().GetBool("verbose")
+	if err != nil {
+		log.Panicln(err)
```

Comment:
I personally would just return the err here but of course that is a place where a panic is also apporpriate